pallets/subtensor/src/staking.rs

```diff
@@ -781,6 +788,31 @@ impl<T: Config> Pallet<T> {
         Ok(credit)
     }
 
+    pub fn kill_coldkey_account(
+        coldkey: &T::AccountId,
+        amount: <<T as Config>::Currency as fungible::Inspect<<T as system::Config>::AccountId>>::Balance,
+    ) -> Result<u64, DispatchError> {
+        if amount == 0 {
+            return Ok(0);
+        }
+
+        let credit = T::Currency::withdraw(
```

Comment:
Are we certain that `withdraw` returns the remaining balance and not the total amount withdrawn here?